scanner.js

```diff
@@ -30,9 +51,15 @@ var dive = function(dir, action) {
       console.log("SKIPPING FILE: Could not stat " + fullpath);
     }
     if(stat && stat.isDirectory()) {
-      dive(fullpath, action);
+      try {
+        dive(fullpath, action);
+      } catch (e) {
+          console.log("SKIPPING FILE: Couldn't parse " + fullpath);
+      }
```

Comment:
I closed #171, to take the patch from here. I like that you kept the consistent upper case `SKIPPING FILE` string. :+1: